tools/docker-deploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+docker pull ubuntu:xenial
+echo -e "\nStarting container..."
+docker run -d --name allinone --privileged -v /sys/fs/cgroup:/sys/fs/cgroup:ro ubuntu:xenial systemd
+echo -e "\nUpdating APT cache..."
+docker exec -i allinone apt-get -qq update
+echo "Installing dependencies..."
+docker exec -i allinone apt-get -qq install python-apt ca-certificates apt-transport-https sudo ssh > /dev/null
+echo -e "\nRunning ansible deployment..."
+ansible-playbook -i inventory/allinone install-ci.yml tests/files/validate-ci.yml -c docker -vv -e @secrets.yml.example --skip-tags monitoring,not-on-docker,letsencrypt
```

Comment:
I'm inclined to either leave this script as it is for now, or maybe throw a `$@` on the end for additional arguments. Would you be alright with that for now?

In the future we can turn this into a more flushed out deployment tool...but I think we'd be better off documenting the steps to manually deploy and test changes for now.